Tests/feaLib/builder_test.py

```diff
@@ -259,7 +260,7 @@ def test_singleSubst_multipleSubstitutionsForSameGlyph(self):
 
     def test_singlePos_redefinition(self):
         self.assertRaisesRegex(
-            FeatureLibError,
+            OTLLibError,
```

Comment:
I find it problematic that we are changing the class of the raised exceptions from FeatureLibError to OTLLibError (which are unrelated to each other). Now all clients (e.g. ufo2ft) will have to catch both, instead of just FeatureLibError. I think feaLib should catch OTLLibError and re-raise it as FeatureLibError to not break current API.